css/styles.css

```diff
@@ -89,3 +94,39 @@ li {
 h2 {
   text-align: center;
 }
+
+.currently-airing {
+  background-color: #1bb55c;
+  height: 4.2rem;
+  width: 11.5rem;
+  border-radius: 0.5rem;
+  margin-top: 3rem;
+  font-size: 1.3rem;
+  padding-bottom: 0.7rem;
+}
+
+.up-and-coming {
+  background-color: #007bff;
+  height: 4.2rem;
+  width: 11.5rem;
+  border-radius: 0.5rem;
+  margin-top: 3rem;
+  margin-left: 5rem;
+  margin-right: 5rem;
+  font-size: 1.3rem;
+  padding-bottom: 0.7rem;
+}
+
+.by-popularity {
+  background-color: #562b81;
+  height: 4.2rem;
+  width: 11.5rem;
+  border-radius: 0.5rem;
+  margin-top: 3rem;
+  font-size: 1.3rem;
+  padding-bottom: 0.7rem;
+}
```

Comment:
You repeat a lot of CSS between these 3 different selectors. I recommend creating a new class with all the css that gets applied to each of these, and giving that class to these 3 elements.